src/lib/swapper/swappers/utils/filterAssetIdsBySellable/filterAssetIdsBySellable.ts

```diff
@@ -1,13 +1,12 @@
-import type { EvmChainId } from '@shapeshiftoss/chain-adapters'
-import { evmChainIds } from '@shapeshiftoss/chain-adapters'
+import { arbitrumNovaChainId } from '@shapeshiftoss/caip'
+import { isEvmChainId } from '@shapeshiftoss/chain-adapters'
 import type { Asset } from '@shapeshiftoss/types'
 
-// we dont perform a lookup to lifi's supported assets because they support far more assets than we do
-// so the overhead in performing the fetch to lifi isnt worth the time
 export function filterEvmAssetIdsBySellable(assets: Asset[]): Asset[] {
   return assets.filter(asset => {
     const { chainId } = asset
 
-    return evmChainIds.includes(chainId as EvmChainId)
+    // evm only with no arbitrum nova support for any swappers
```

Comment:
This isn't working as it should - Nova is still available as a buy and sell asset

<img width="456" alt="image" src="https://github.com/shapeshift/web/assets/17035424/aee3a367-ab82-480e-9a7b-2c71454691f5">
<img width="457" alt="image" src="https://github.com/shapeshift/web/assets/17035424/f7b04bc1-0656-42c7-949e-957352d0322e">

Reply:
Cache clear fixed as per dms